app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= provide :title, "Help" %>
```

Comment:
translate all text in this file using I18n (the same as `app/views/layouts/_header.html.erb` file)